app/views/pets/index.html.erb

```diff
@@ -1,3 +1,6 @@
+<%= render 'header' %>
```

Comment:
Nice job getting a partial. Would love to see you continue to practice with forms